theme/static/js/vue-2.6.10-min.js

```diff
@@ -0,0 +1,6 @@
+/*!
```

Comment:
If we are switching to the minified file of Vue, consider removing the dev file we've been serving in other pages.

Reply:
I'd like to take a comprehensive approach with webpack or other proper approach, outside this PR. All files should be dev version and then a consolidation command run as a part of "compilation"/transpile. We can wrap this in with the ES6 work...

Reply:
I mean, we are already serving another Vue file. This PR is adding the same file again, but minified. We should not be serving both.

Reply:
I can remove the minified version.